tools/traffic/table/blob_metadata.go

```diff
@@ -0,0 +1,69 @@
+package table
+
+import "errors"
+
+// BlobMetadata encapsulates various information about a blob written by the traffic generator.
+type BlobMetadata struct {
+	// key of the blob, set when the blob is initially uploaded.
+	key []byte
```

Comment:
Is this the blobKey: https://github.com/Layr-Labs/eigenda/blob/2566c2135b5174c23560fcbe425b1aec01b8df73/disperser/disperser.go#L52 ?

Reply:
The key, in this context, is the `[]byte` return value of this method:

```
func (c *disperserClient) DisperseBlob(ctx context.Context, data []byte, quorums []uint8) (*disperser.BlobStatus, []byte, error) {
```

The `BlobKey` seems like it is holding the same data, although it is in string form, not byte array form. I originally had the `MetadataHash` field as well, although that is currently removed in this PR.

Should I be using the `BlobKey` struct here?